pkg/tasks/authorization.go

```diff
@@ -0,0 +1,76 @@
+package tasks
+
+import (
+	"context"
+
+	cs "github.com/cloudtrust/common-service"
+	"github.com/cloudtrust/common-service/log"
+	"github.com/cloudtrust/common-service/security"
+	api "github.com/cloudtrust/keycloak-bridge/api/tasks"
+)
+
+var actions []security.Action
+
+func newAction(as string, scope security.Scope) security.Action {
+	a := security.Action{
+		Name:  as,
+		Scope: scope,
+	}
+
+	actions = append(actions, a)
+	return a
+}
+
+// Actions used for authorization module
```

Comment:
Do we need to put authorization layer with rights for this API ?

Reply:
Discussed together by phone...